README.md

```diff
@@ -406,13 +406,14 @@ description: A cool pipeline
 author: me
 prefix: cool-pipes-company
 skip:
+  - github
   - ci
   - github_badges
   - igenomes
   - nf_core_configs
 ```
 
-This will create a pipeline called `cool-pipe` in the directory `cool-pipes-company-cool-pipe` with `me` as the author. It will exclude the GitHub CI from the pipeline, remove GitHub badges from the `README.md` file, remove pipeline options related to iGenomes and exclude `nf_core/configs` options.
+This will create a pipeline called `cool-pipe` in the directory `cool-pipes-company-cool-pipe` with `me` as the author. It will exclude all files required for GitHub hosting of the pipeline, the GitHub CI from the pipeline, remove GitHub badges from the `README.md` file, remove pipeline options related to iGenomes and exclude `nf_core/configs` options.
 
```

Comment:
Something like this? what do you think?
```suggestion
Selecting `github` to remove all GitHub support will also exclude `ci` and `github_badges` by default.

```

Reply:
where is the logic that couples `github_badges`+`ci` to `github`?

Reply:
`github` removes the whole `.github` folder, which includes `.github/workflows/` for CI. But for `github_badges` I think you are right, and I need to add an extra check.